components/org.wso2.carbon.identity.governance/src/main/java/org/wso2/carbon/identity/governance/IdentityMgtConstants.java

```diff
@@ -26,6 +26,29 @@ private IdentityMgtConstants(){}
     public static final String USER_IDENTITY_CLAIMS = "UserIdentityClaims";
     public static final String ERROR_CODE_DELIMITER = "-";
 
+    /**.
+     * DataType Constants
```

Comment:
Add a meaningful full comment. The best practice is to start the comment with an upper case letter and end with a `.`

Reply:
Fixed in commit 8032899333352cf8a310623b49856dfb99ec9767